### PR TITLE
@W-10361847 - Replace LWC Services

### PR DESCRIPTION
**Pull request by lozforce**

### CODE REVIEW

File: packages/client/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        new LwcWebpackPlugin({
+            // https://www.npmjs.com/package/lwc-webpack-plugin
+            modules: [
+                {
+                    dir: 'src/modules'
+                },
+                {
+                    npm: 'lightning-base-components'
+                }
+            ]
```
Comment (nolanlawson) [inline packages/client/webpack.config.js:48]: FWIW, you can keep your `lwc.config.json` and lwc-webpack-plugin will automatically pick it up.

File: packages/client/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                    '..',
+                    'node_modules',
+                    '@salesforce-ux',
+                    'design-system',
+                    'assets'
+                ),
+                to: path.join(buildDir, 'assets')
+            }
+        ]),
+        new ErrorOverlayPlugin()
```
Comment (nolanlawson) [inline packages/client/webpack.config.js:71]: You might only want this in dev mode.
```suggestion
        ...(mode === 'development' ? [new ErrorOverlayPlugin()] : [])
```

**Review by nolanlawson — APPROVED** [review]
LGTM!

**Review by tariqrafique — APPROVED** [review]

File: packages/client/webpack.config.js (hunk 1 above)
Comment (lozforce) [reply packages/client/webpack.config.js:48]: Good to know. Having fewer config files is more manageable.

File: packages/client/webpack.config.js (hunk 2 above)
Comment (lozforce) [reply packages/client/webpack.config.js:71]: Good idea

**Pull request merged**